File: problems/prob35/prob35.py
def ex_oh(string):
    ex = []
    oh = []
    for char in string:
        if char == 'x':
            ex.append(char)
        else:
            oh.append(char)
    if len(ex) == len(oh):
        return True
    else:
        return False


print ex_oh('xooxxo') == True
print ex_oh('x') == False

File: problems/prob35/prob35.rb
def ex_oh(string)
  ex = []
  oh = []
  string.each { |char| char == 'x' ? ex << char : oh << char }
  ex.length == oh.length ? true : false
end


puts ex_oh(('xooxxo').split("")) == true
puts ex_oh(('x').split("")) == false
puts ex_oh(('xoxo').split("")) == true